Src/ILGPU/Runtime/ExchangeBuffer.cs

```diff
@@ -177,17 +179,33 @@ protected override void Dispose(bool disposing)
         /// </summary>
         public TIndex Extent { get; protected set; }
 
+        /// <summary>
+        /// Internal array view
+        /// </summary>
+        [SuppressMessage(
+            "Microsoft.Design",
+            "CA1051: Do not declare visible instance fields",
+            Target = "target")]
```

Comment:
ISSUE: Remove `Target = "target"`. This is used for methods that have multiple parameters, and you want to pick which parameter to suppress.